Orm/Xtensive.Orm/Orm/Linq/Expressions/FieldExpression.cs

```diff
@@ -119,9 +119,12 @@ public virtual FieldExpression RemoveOwner()
 
     public static FieldExpression CreateField(FieldInfo field, int offset)
     {
-      if (!field.IsPrimitive)
+      if (!field.IsPrimitive) {
         throw new ArgumentException(string.Format(Strings.ExFieldXIsNotPrimitive, field.Name), "field");
-      var mapping = new Segment<int>(field.MappingInfo.Offset + offset, field.MappingInfo.Length);
+      }

```

Comment:
I guess need to replace ```"field"``` with ```nameof(field)``` as you did in other similar cases.

Reply:
I've changed this.